aspnetcore/security/anti-request-forgery/sample/MvcSample/MvcSample.csproj

```diff
@@ -1,22 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
Consider upgrading to 2.1.

Reply:
It's on the list. I didn't mark myself for it; but if I have time, I'll do it. https://github.com/aspnet/Docs/issues/5495